pkg/operators/machine-api-operator.go

```diff
@@ -28,22 +28,22 @@ var _ = Describe("[Feature:Operators] Machine API operator deployment should", f
 		Expect(err).NotTo(HaveOccurred())
 
 		deploymentName := "machine-api-controllers"
-		initialDeployment, err := framework.GetDeployment(client, deploymentName)
+		initialDeployment, err := framework.GetDeployment(client, deploymentName, framework.MachineAPINamespace)
 		if err != nil {
-			initialDeployment, err = framework.GetDeployment(client, deploymentDeprecatedName)
+			initialDeployment, err = framework.GetDeployment(client, deploymentDeprecatedName, "")
```

Comment:
```suggestion
			initialDeployment, err = framework.GetDeployment(client, deploymentDeprecatedName, framework.MachineAPINamespace)
```

Reply:
thanks! I actually dropped the deprecated deployment check now https://github.com/openshift/cluster-api-actuator-pkg/pull/152/commits/f19ea2894ea68272a794a6b9aa3ca817acb6e751